app/controllers/sufia/batch_uploads_controller.rb

```diff
@@ -19,6 +19,7 @@ def create_update_job
 
     def build_form
       @form = form_class.new(curation_concern, current_ability)
+      @form.action_name = action_name
```

Comment:
again, good change here I think - no need to do all the overriding of constructors in subclasses if all we want to do is send a message to the object.